firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -49,37 +49,60 @@ void disableMotors()
 // Drives the motor corresponding to "motor" to it's home position (position 0)
 void homeAxis(int motor[])
 {
+  int * currentMotorPos;
+  int currentMotorDir, currentMotorNumEighthSteps;
   int i;
 
+  //  Stores corresponding motor position based off of which motor is being homed,
```

Comment:
Sorry for the additional comment - there should be one space before the start of the comment here instead of two